File: python/Course.py
class Course:
# kelas yang digunakan untuk mengimplentasikan kelas Course

    ## inisialiasasi attribute
    # __nama_matakuliah =""

    # konstruktor
    def __init__(self, nama_matakuliah):
        self.__nama_matakuliah = nama_matakuliah

    # setter & setter methods #

    def setNamaMataKuliah(self, nama_matakuliah):
    # mengset nilai atribut nama
        self.__nama_matakuliah = nama_matakuliah

    def getNamaMataKuliah(self):
    # mengembalikan nilai atribut nama_matakuliah
        return self.__nama_matakuliah

File: c++/Course.cpp
class Course {
    private:
        /* data */
        string nama_matakuliah;
    public:
        Course() {};
        Course(string namaMatkul) {
            this->nama_matakuliah = namaMatkul;
        }

        void setNamaMatkul(string namaMatkul) {
            this->nama_matakuliah = namaMatkul;
        }

        string getNamaMatkul() {
            return nama_matakuliah;
        }
        ~Course(){}
};
